lib/auth/webauthnwin/api.go

```diff
@@ -188,7 +189,12 @@ type CheckSupportResult struct {
 // false positives.
 // See CheckSupport.
 func IsAvailable() bool {
-	return CheckSupport().IsAvailable
+	supports := CheckSupport()
+	if supports.HasCompileSupport && !supports.IsAvailable {
+		log.Warn("Webauthn is not supported on this version of Windows, supported from version 1903")
```

Comment:
```suggestion
		log.Warn("Webauthn is not supported on this version of Windows, it requires Windows 10 version 1903 or newer")
```

Right?

Reply:
we just learned that webauthn.dll is also present on windows server. And 1903 is valid version of windows server:
> Windows Server, version 1903 (unsupported as of December 8, 2020; 2 years ago)

so maybe we can leave it without windows10 info.

Reply:
Could we say "Windows 10 or Windows Server, version 1903"? Would that be correct?

Feel free to ignore if you think that makes is worse.